cmd/entrypoint/namespaces_test.go

```diff
@@ -15,6 +15,18 @@ import (
 // sandboxed environment, the test could pass even if the dropNetworking
 // function did nothing.
 func TestDropNetworking(t *testing.T) {
+	// First make sure we can run the dropNetworking command.
+	// Some older kernels require special configurations to run this.
+	// I haven't been able to come up with an exhaustive list of what is needed,
+	// but it includes things like CAP_SYS_ADMIN, kernel.unprivileged_userns_clone=1
+	// and maybe others.
+	// For the sake of this test just check it first.
+	testCmd := exec.Command("true")
+	dropNetworking(testCmd)
+	if _, err := testCmd.CombinedOutput(); err != nil {
+		t.Skipf("skipping test as required namespace features are not available: %v", err)
```

Comment:
Would it make sense to also include the combined stdout/stderr here too?

Reply:
Nothing shows up in the output when I get an error. I'm not sure if that's always the case, though:

```
    namespaces_test.go:27: skipping test as required namespace features are not available: fork/exec /bin/true: operation not permitted
```